src/ActionSheet/TouchableNativeFeedbackSafe.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import {
+  Platform,
+  TouchableNativeFeedback,
+  TouchableOpacity,
+  View,
+  TouchableWithoutFeedbackProps,
+} from 'react-native';
+
+// This TouchableOpacity has the same staic method of TouchableNativeFeedback
+class CustomTouchableOpacity extends TouchableOpacity {
```

Comment:
do we want to extend TouchableOpacity here? Is this fair game @brentvatne ?

Reply:
The original code is here
https://github.com/expo/react-native-action-sheet/blob/5ee7d9af74f3272deceb9b62a5035dbd9c3eea0a/ActionSheet.js#L424-L433

Add static method for TouchableOpacity